scripts/update-drafts.ts

```diff
@@ -137,12 +138,29 @@ async function main() {
       continue;
     }
 
-    const spec_url = feature.data.__compat.spec_url;
-    if (!spec_url) {
-      continue;
+    let parent_url = null;
+    if (!feature.data.__compat.spec_url) {
+      const parent_urls = [];
+      const path = feature.id.split(".");
+      let parentFeature = compat.data as Identifier;
+      while (path.length > 1) {
+        parentFeature = parentFeature[path.shift()];
+        const parent_spec = parentFeature.__compat?.spec_url;
```

Comment:
Similarly, you can use `compat.features` to get an object with a `spec_url` getter (probably):

```suggestion
      while (path.length > 1) {
        const parent_spec = compat.features.get(path.shift())?.spec_url ?? [];
```

Reply:
This didn't quite work- it was calling `compat.features.get('properties')` instead of `compat.features.get('css.properties')`. I did switch to building a path up and using `compat.features.get()` instead of drilling through the raw data.